src/libs/OptionsListUtils.js

```diff
@@ -439,7 +440,14 @@ function getOptions(reports, personalDetails, activeReportID, {
             return;
         }
 
-        if (ReportUtils.isDefaultRoom(report) && !Permissions.canUseDefaultRooms(betas)) {
+        // If one is a member of a free policy, then they are allowed to see the Policy default rooms.
+        // For everyone else, one must be on the beta to see a default room.
+        const isMemberOfFreePolicy = PolicyUtils.isMemberOfFreePolicy(policies);
+        if (isMemberOfFreePolicy && !Permissions.canUseDefaultRooms(this.props.betas)) {
```

Comment:
```suggestion
        if (isMemberOfFreePolicy && !Permissions.canUseDefaultRooms(betas)) {
```

`this` is `undefined` here

<img width="701" alt="image" src="https://user-images.githubusercontent.com/87341702/173699543-5ee1f93a-6225-4941-90ad-b5c373c1899c.png">

Reply:
Oops yes sorry I had fixed this but forgot to force-push the update yesterday!